lib/Traits/WebMapServiceCatalogItemTraits.ts

```diff
@@ -259,4 +259,35 @@ export default class WebMapServiceCatalogItemTraits extends mixTraits(
       "Gets or sets the coverage name for linked WCS for clip-and-ship."
   })
   linkedWcsCoverage?: string;
+
+  @primitiveTrait({
+    type: "string",
+    name: "Is Thredds",
+    description: "True if this WMS is from a THREDDS server; otherwise, false."
+  })
+  isThredds: boolean = false;
```

Comment:
I always get confused whether things should be optional if they have a default, eg 
````
  isThredds?: boolean = false;
````